src/api.ts

```diff
@@ -66,7 +66,8 @@ export const request = (
   retryNumber: number = 0,
   options: RequestOptions = {}
 ): Promise<any> => {
-  let body = data && { ...data };
+  const body = data instanceof Array ? data && [...data] : data && { ...data };
```

Comment:
I think you can't do that because if `data` is `undefined`  `{ ...data }` will failed.